.github/workflow/linters.yml

```diff
@@ -0,0 +1,56 @@
+name: Linters
+
+on: pull_request
+
+env:
+  FORCE_COLOR: 1
+
+jobs:
+  lighthouse:
+    name: Lighthouse
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup Lighthouse
+        run: npm install -g @lhci/cli@0.7.x
+      - name: Lighthouse Report
+        run: lhci autorun --upload.target=temporary-public-storage --collect.staticDistDir=.
+  webhint:
+    name: Webhint
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup Webhint
+        run: |
+          npm install --save-dev hint@7.x
+          [ -f .hintrc ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css/.hintrc
+      - name: Webhint Report
+        run: npx hint .
+  stylelint:
+    name: Stylelint
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup Stylelint
+        run: |
+          npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+          [ -f .stylelintrc.json ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css/.stylelintrc.json
+      - name: Stylelint Report
+        run: npx stylelint "**/*.{css,scss}"
+  nodechecker:
+    name: node_modules checker
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: Check node_modules existence
+        run: |
+          if [ -d "node_modules/" ]; then echo -e "\e[1;31mThe node_modules/ folder was pushed to the repo. Please remove it from the GitHub repository and try again."; echo -e "\e[1;32mYou can set up a .gitignore file with this folder included on it to prevent this from happening in the future." && exit 1; fi
```

Comment:
- Good job setting up linters :+1: but it's not working at the moment. Here is a screenshot to prove that :point_down: 

![image](https://user-images.githubusercontent.com/99715640/219543026-5bbfcd8c-8cf5-48cb-b021-96ea50b493aa.png)

  - [ ] Kindly set up linter correctly in this repository, and its result should be green, this means that the last commit should have a green check mark (:heavy_check_mark: ) or a summary of the passed checks is in the PR.

- Maybe you can start by checking the name of this folder because noticed that your `workflow` is missing a `s`. Kindly rename it to `workflows` and see if it works.

_**Linter is an industry-standard tool that is used by organizations to uniformly ensure that codes are well structured/writing for maintainability and promote uniform syntax and coding styles among myriads of developers :wink: and for you as a developer, it helps you to become aware of those practices that can distinct you.**_ :hammer_and_wrench:

Reply:
Thanks. I'll gonna fix it today!

Reply:
- [ ]  Please set up your linters & GitHub workflow properly. Lintes help us write uniform code of the same quality. use this [resource](https://github.com/microverseinc/linters-config/tree/master/html-css) 😆

Reply:
- [OPTIONAL] It would be very nice you resolve the conflict between your branches 🙏🏾